src/main/java/io/github/hypixel_api_wrapper/wrapper/player/HypixelPlayerStats.java

```diff
@@ -0,0 +1,24 @@
+package io.github.hypixel_api_wrapper.wrapper.player;
+
+import io.github.hypixel_api_wrapper.wrapper.hypixel_object_factory.HypixelBedWarsFactory;
+import java.util.Optional;
+import org.json.JSONObject;
+
+/**
+ * Used to keep track of a {@link HypixelPlayer}'s statistics on the
+ * Hypixel Network.
+ */
+public class HypixelPlayerStats {
+
+    private final JSONObject stats;
+    private HypixelBedWarsFactory factory;
+
+    public HypixelBedWarsFactory getBedWarsStats() {
+        return Optional.ofNullable(factory).orElse(factory = new HypixelBedWarsFactory(stats));
+    }
+
+    public HypixelPlayerStats(JSONObject stats) {
+        this.stats = stats;
+    }
```

Comment:
It's conventional to put constructors before methods, that way it's clearer how a class's fields are initialized
```suggestion
    public HypixelPlayerStats(JSONObject stats) {
        this.stats = stats;
    }

    public HypixelBedWarsFactory getBedWarsStats() {
        return Optional.ofNullable(factory).orElse(factory = new HypixelBedWarsFactory(stats));
    }
```